package.json

```diff
@@ -57,6 +57,7 @@
     "eslint-plugin-storybook": "^0.6.14",
     "jest": "^27.4.5",
     "jest-cli": "^27.4.5",
+    "lit": "^3.1.2",
```

Comment:
```suggestion
```

Reply:
"Iit" is used for storybook to render the components.
I added it as dev dependency

```
import { html } from 'lit';
...
const meta: Meta<HTMLGxQueryViewerControllerElement> = {
  component: "gx-query-viewer-controller",
  decorators: [(story, context) => html`<gx-query-viewer type="${context.args.type}">${story()}</gx-query-viewer>`],

```